crates/telio-model/src/mesh.rs

```diff
@@ -90,6 +92,16 @@ pub enum NodeState {
     Connected,
 }
 
+/// This is a hint state computed based on the last_rx_timestamp
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LinkState {
+    /// Link is Down
+    Down,
+    /// Link is Up
+    Up,
```

Comment:
nitpick: these are obvious because the names are good

Reply:
I am forced to add documentation :(

Reply:
oh..